modules/exploits/multi/http/gitlab_github_import_rce_cve_2022_2992.rb

```diff
@@ -0,0 +1,242 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  include Msf::Exploit::Git::SmartHttp
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
+  include Msf::Exploit::Remote::HTTP::Gitlab
+  include Msf::Exploit::RubyDeserialization
+
+  attr_accessor :cookie
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Remote Command Execution via Github import',
```

Comment:
We try to give the module title the same or similar name to the name of the file. That said, both could be more descriptive of the vulnerability. Maybe something akin to `Gitlab Github Repo Import Deserialization`

Reply:
Changed name to `Gitlab Github Repo Import Deserialization RCE`.